### DISCUSSION TITLE
Best plugin of all time

### DISCUSSION THREAD

**Discussion by m4q** [ftolsson/ftoutliner#2] [General]
Never stop supporting it, please.

**Comment by m4q** [comment]
Also, could you complete the wiki, pretty please?

**Reply by ftolsson** [reply]
Thank you so much! And yes, your right — I really should. I'd actually somewhat forgotten about it, I'll try to find some time for getting it into shape!

**Reply by Mykbibby** [reply]
Please complete the wiki!!
